starter-code/javascript/main.js

```diff
@@ -7,54 +7,86 @@ var secDec      = document.getElementById('secDec');
 var secUni      = document.getElementById('secUni');
 var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
+var list        = document.getElementById('splits');
 
+var minutes = chronometer.twoDigitsNumber(minutes);
+var seconds = chronometer.twoDigitsNumber(seconds);
 
-function printTime() {
-
-}
-
-function printMinutes() {
-
+    
+function printMinutes(){
+    minDec.innerHTML = minutes[0];
+    minUni.innerHTML = minutes[1];
 }
 
 function printSeconds() {
-
+    secDec.innerHTML = seconds[0];
+    secUni.innerHTML = seconds[1];
 }
 
-function printMilliseconds() {
-
+function printTime() {
+    printMinutes();
+    printSeconds();
 }
 
 function printSplit() {
-
+    var split = document.createElement("li");
+    split.innerHTML = minutes + " : " + seconds ;
+    split.style.display = "block";
+    list.appendChild(split);
 }
 
 function clearSplits() {
+    list.removeChild(childNodes[0]);
+}
 
+function setStartBtn() {
+    btnLeft.classList.add("start");
+    btnLeft.classList.remove("stop");
+    btnLeft.innerHTML="START";
 }
 
 function setStopBtn() {
-
+    btnLeft.classList.add("stop") ;
+    btnLeft.classList.remove("start") ;
+    btnLeft.innerHTML ="STOP";
 }
 
 function setSplitBtn() {
-
+    btnRight.classList.add("split");
+    btnRight.classList.remove("reset");
+    btnRight.innerHTML="SPLIT";
 }
 
-function setStartBtn() {
-
+function setResetBtn() {
+    btnRight.classList.add("reset");
+    btnRight.classList.remove("split");
+    btnRight.innerHTML="RESET";
 }
 
-function setResetBtn() {
 
-}
 
 // Start/Stop Button
-btnLeft.addEventListener('click', function () {
 
+btnLeft.addEventListener('click', function () {
+    console.log(btnLeft.innerHTML);
+    if (btnLeft.innerHTML === "START") {    
+        chronometer.startClick();
+        printTime();
+        setStopBtn();
+    } else {    
+        chronometer.stopClick();
+        setStartBtn();
+    }     
```

Comment:
For me, it seems that the counter itself it's working great, but for some reason, it's not printing. I've tried to check through different ways but couldn't find where does the mistake is.